integration_tests/dbt_project.yml

```diff
@@ -1,5 +1,5 @@
 name: 'quickbooks_source_integration_tests'
-version: '0.6.0'
+version: '0.6.1'
```

Comment:
This should be a breaking change since it is adding quite a bit more with the union feature and what not.

Reply:
Good call, updating.